org.adempiere.base/src/org/compiere/model/MLookupFactory.java

```diff
@@ -1024,9 +1024,9 @@ static public String getLookup_TableDirEmbed (Language language,
 		//
 		StringBuilder embedSQL = new StringBuilder("SELECT ");
 
-		StringBuilder displayColumn = getDisplayColumn(language, TableName, list, BaseTable);
+		StringBuilder displayColumn = getDisplayColumn(language, "alias", list, BaseTable);
 		embedSQL.append(displayColumn.toString());
-		embedSQL.append(" FROM ").append(TableName);
+		embedSQL.append(" FROM ").append(TableName).append(" alias");

```

Comment:
Please check comment at:
https://idempiere.atlassian.net/browse/IDEMPIERE-6152?focusedCommentId=50872

Also, it's possible that "alias" is a [reserved word in some databases](https://www.drupal.org/docs/develop/coding-standards/list-of-sql-reserved-words), better to use a different name